Detectors/TPC/reconstruction/include/TPCReconstruction/KrBoxClusterFinder.h

```diff
@@ -134,20 +154,23 @@ class KrBoxClusterFinder
   float mQThreshold = 1.0;        ///< every charge which is added to a cluster must exceed this value or it is discarded
   int mMinNumberOfNeighbours = 1; ///< amount of direct neighbours required for a cluster maximum
 
+  int mSector = -1;                 ///< sector being processed in this instance
   std::unique_ptr<CalPad> mGainMap; ///< Gain map object
 
   /// Maximum Map Dimensions
   /// Here is room for improvements
-  static constexpr size_t MaxPads = 138;  ///< Size of the map in pad-direction
-  static constexpr size_t MaxRows = 152;  ///< Size of the map in row-direction
-  static constexpr size_t MaxTimes = 550; ///< Size of the map in time-direction
+  static constexpr size_t MaxPads = 138;    ///< Size of the map in pad-direction
+  static constexpr size_t MaxRows = 152;    ///< Size of the map in row-direction
+  static constexpr size_t MaxTimes = 11000; ///< Size of the map in time-direction
```

Comment:
Are 11000 time bins sufficient?
E.g. in the run taken at 98% of the gain, the timeStamp exceeds 11000 (it is close to 14000).

Reply:
Good point. I can change it, but it requires some better treatment in future. Otherwise we'll at some point fill up all the memory ..

Reply:
Sure, we discussed potential solutions. I will think about an elegant way to implement them.